docs/en/05_Contributing/04_Release_Process.md

```diff
@@ -33,34 +36,98 @@ announced on the ["releases" forum category](https://forum.silverstripe.org/c/re
 
 ## Release numbering {#release-numbering}
 
-SilverStripe follows [Semantic Versioning](http://semver.org).
+Silverstripe CMS follows [Semantic Versioning](http://semver.org).
 
-All SilverStripe modules (including silverstripe/framework) may have patch releases created at any time, and will
+All Silverstripe CMS modules (including `silverstripe/framework`) may have patch releases created at any time, and will
 not necessarily match other core module patch release numbers. For example, your project may be using
-silverstripe/cms 4.3.1 with silverstripe/versioned 4.3.9.
+`silverstripe/cms` 4.3.1 with `silverstripe/versioned` 4.3.9.
 
-All SilverStripe recipes are released in lock step with each other. For example, silverstripe/installer 4.3.1 will
-contain silverstripe/recipe-cms 4.3.1 and silverstripe/recipe-core 4.3.1. These recipes may contain various patch
+All Silverstripe CMS recipes are released in lock step with each other. For example, `silverstripe/installer` 4.3.1 will
+contain `silverstripe/recipe-cms` 4.3.1 and `silverstripe/recipe-core` 4.3.1. These recipes may contain various patch
 versions of its modules within the same minor release line (4.3 in this example).
 
+## Regular quarterly releases
+
+A regular Silverstripe CMS minor release is tagged every quarter. This involves creating a new minor increment of 
+each core recipe. Regular quarterly releases are preceded by pre-stable releases to help lay the groundwork for the
+stable release.
+
+### Beta phase
+
+At the start of the Beta phase:
+* new minor branches for core modules are forked from the default branches
+* a feature freeze is instituted on the minor branches
+* a `beta1` release is tagged for recipes and related modules.
+
+The beta phase lasts approximately one month. Additional betas may be released to address critical issues holding back
+testers.
+
+#### "Beta release" definition
+
+The purpose of a beta release is to stabilise the Silverstripe CMS codebase for an upcoming stable release and showcase 
+new features. A beta release is likely to contain regressions and new bugs. A beta release is primarily aimed at testers 
+who want to help identify bugs and regressions.
```

Comment:
The RC phase section mentions releasing a public RC tag. Assuming you'll do this for betas too, do you want to list that here as well?

Reply:
The point of mentioning the public RC tag was to differentiate from the private RC we send to the auditors. I'm happy to change the point above to "a publicly visible `beta1` release is tagged for recipes and related modules" if we feel this is distinction is not clear.